com.app.NoteSlayer.yaml

```diff
@@ -0,0 +1,41 @@
+app-id: com.app.NoteSlayer
```

Comment:
you don't own `app.com`

This isn't what I suggested. One doesn't use a random domain they feel like it.

Reply:
this is the domain for the app I bought 
https://note-slayer.com/

Reply:
This is what I wrote:

https://github.com/flathub/flathub/pull/4616/files/f0fdd8058142823a77ee23c5b9810ce7b4e26ef1#diff-47f747f676fbab26edae82a9f55df305401b243cc6a778ba6751b558adcb7529

![image](https://github.com/flathub/flathub/assets/114441/f07dd475-7eb9-40d6-b480-d92c638796ee)